source/server/hot_restarting_child.cc

```diff
@@ -172,5 +217,42 @@ void HotRestartingChild::mergeParentStats(Stats::Store& stats_store,
   stat_merger_->mergeStats(stats_proto.counter_deltas(), stats_proto.gauges(), dynamics);
 }
 
+void HotRestartingChild::onSocketEventUdpForwarding() {
+  std::unique_ptr<HotRestartMessage> wrapped_request;
+  while ((wrapped_request =
+              udp_forwarding_rpc_stream_.receiveHotRestartMessage(RpcStream::Blocking::No))) {
+    if (wrapped_request->requestreply_case() == HotRestartMessage::kReply) {
+      ENVOY_LOG(
```

Comment:
Use `ENVOY_LOG_PERIODIC` or similar; if this prints, it could blow up the process with logging because it will probably happen A LOT if it happens at all.

Reply:
Done.